dirty.rb

```diff
@@ -0,0 +1,34 @@
+a =     'b'
+eval '1+1'
+"#{1}"
```

Comment:
__[development][rubocop]__
* Literal `"#{1}"` used in void context.
* Prefer `to_s` over string interpolation.
* Literal interpolation detected.